hack/lib/catalogsource.bash

```diff
@@ -80,7 +80,7 @@ spec:
         - |-
           podman login -u $pull_user -p $token image-registry.openshift-image-registry.svc:5000 && \
           /bin/opm registry add                         -d index.db --mode=replaces -b docker.io/markusthoemmes/serverless-bundle:1.7.2 && \
```

Comment:
@markusthoemmes or @warrenvw 

Do we have a new/different image?  Is there a repo? Curious, b/c of "markusthoemmes" docker.io account :smile_cat:

Reply:
I guess @warrenvw's work in https://github.com/openshift-knative/serverless-operator/pull/414 would replace this.